src/__stories__/logo-story.tsx

```diff
@@ -1,82 +1,98 @@
 import * as React from 'react';
 import {BlauLogo, Logo, MovistarLogo, O2Logo, TelefonicaLogo, VivoLogo} from '../logo';
-import {StorySection} from './helpers';
-import {Text2} from '../text';
-import {ThemeVariant} from '../theme-variant-context';
-import {skinVars} from '../index';
+import {Box, ResponsiveLayout} from '../index';
 
 export default {
     title: 'Components/Logo',
     component: Logo,
     parameters: {
         fullScreen: true,
     },
-    inverse: false,
 };
 
-type Args = {type: 'isotype' | 'imagotype' | 'vertical'; size: number; inverse: boolean};
+type Args = {
+    type: 'isotype' | 'imagotype' | 'vertical';
+    size: number;
+    inverse: boolean;
+    action: string;
+    forceBrandLogo: boolean;
+    brand: 'Movistar' | 'O2' | 'Vivo' | 'Vivo-new' | 'Telefonica' | 'Blau';
```

Comment:
you can remove `Vivo-new` from this selector, the brand for Vivo and Vivo-new skin is Vivo :)

Reply:
Good catch. Fixed